### DISCUSSION TITLE
Welcome to Godot-SDL-Gyro Discussions!

### DISCUSSION THREAD

**Discussion by SagaPDev** [SagaPDev/Godot-SDL-Gyro#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Make suggestions

**Comment by headblade4865** [comment]
Can you add boolean flag or method for check if gyro calibrated?
eg
```gdscript
func _ready(): 
    Gyro.calibrate()

func _process(): 
    is_gyro_calibrated: bool = Gyro.is_gyroscope_calibrated
    if is_gyro_calibrated:
        Gyro.stop_calibrate() # This is only if necessary. Alternatively, it might be possible to make the calibration stop automatically
        Gyro.gamepad_poiling()
        # Work with gyro values...
```

**Reply by SagaPDev** [reply]
thanks for showing interest but due to the soon to be implemented SDL gyro soon to me merged to Godot itself i doubt i will resume working on this
